sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/SparkMetadataOperationSuite.scala

```diff
@@ -156,6 +156,7 @@ class SparkMetadataOperationSuite extends HiveThriftJdbcTest {
 
     withJdbcStatement("table1", "table2") { statement =>
       Seq(
+        "DROP VIEW IF EXISTS view1",
```

Comment:
Is this change necessary?

Reply:
This view was not cleaned up when the UT failed in my environment. So adding this could stable the test case I think.

Reply:
Weird. If the view is not dropped, why the tables do not exist in your environment?